doc/crud.md

```diff
@@ -234,6 +319,11 @@ CDTDocumentRevision *updated = [datastore updateDocumentFromRevision:retrieved
                                                                error:&error];
 
 ```
+```swift
+let retrieved = try datastore.getDocumentWithId("mydoc")
+retrieved.attachments.removeValueForKey("cute_cat.jpg")
```

Comment:
Is this really meant to be `Value` or should it be `removeObjectForKey`? (because Value in Cocoa is normally for K/V stuff)